packages/typespec-go/src/tcgcadapter/clients.ts

```diff
@@ -710,6 +710,10 @@ export class clientAdapter {
       for (const example of sdkMethod.operation.examples) {
         const goExample = new go.MethodExample(example.name, {summary: example.description}, example.filePath);
         for (const param of example.parameters) {
+          if (param.parameter.isApiVersionParam && param.parameter.clientDefaultValue) {
+            // skip the api-version param as it's not a formal parameter
+            continue;
+          }
```

Comment:
i'm wondering why the subscription id is not impacted, it should also be a client param